src/Symfony/Component/Validator/Tests/Fixtures/Attribute/Entity.php

```diff
@@ -22,6 +22,11 @@
     Assert\GroupSequence(['Foo', 'Entity']),
     Assert\Callback([CallbackClass::class, 'callback']),
 ]
+/**
```

Comment:
Why do we use annotations for the `Sequentially` constraint while using attributes for the rest?

Reply:
This is the fixture for PHP 8.0 where we cannot nest attributes. There is also a fixture `NestedAttribute` that does not use Doctrine-style annotations at all.

Reply:
this Fixture is used for php 8.0. PHP 8.0 does not support nested attributes.

Reply:
Same as for `@All` below, because nested attributes are not available until PHP 8.1?
We could probably add a PHP 8.1 specific test though if not already.

Reply:
ah, got it, thank you 👍